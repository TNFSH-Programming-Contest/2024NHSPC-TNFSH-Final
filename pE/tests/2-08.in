1 4163250 2537579 4866617 8268084
42
