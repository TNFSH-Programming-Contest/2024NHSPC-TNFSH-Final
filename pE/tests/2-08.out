12778090890158550
